### PR TITLE
Game timer draw

### PR DESCRIPTION
**Pull request by PenguinFlyingHigh**
Split flap effect for displaying date, mon, year

### CODE REVIEW

**Comment by PenguinFlyingHigh** [comment]
gametimer

File: src/game_timer.cpp [hunk 1]
```diff
@@ -332,16 +483,110 @@ void GameTimer::SplitSetDateDigits(int day, gt_tracker* gt_day, int mon, gt_trac
...
+
+	}
+	else if (year_d3_flip == 0) {//done flipping, waiting for new flip
+		gt_year->digit_3.old_offset = year_offset_d3;
+	}
 }
 
 void GameTimer::advance_time(float real_time_seconds_elapsed)
 {	
 	const int One_Day_Sec = 86400;
```
Comment (nikrolas) [inline src/game_timer.cpp]: Looking at the calendar, it is going real fast. Do we want Roy's time being a traffic cop to be by year or his whole life? If we want level progression, then it should be by year in my opinion

Comment (PenguinFlyingHigh) [reply src/game_timer.cpp]: We can change the time freely depending on what we decide to increment levels by

**Review by nikrolas — APPROVED** [review]
Looks good

**Pull request merged**